src/braket/pennylane_plugin/translation.py

```diff
@@ -403,6 +416,50 @@ def _(adjoint: Adjoint, parameters):
     return base.adjoint()[0]
 
 
+def translate_parametrized_evolution(
+    op: ParametrizedEvolution, wire_map, device_frames, max_amplitude
+):
+    start, end = op.t[0], op.t[1]
+    pulse_length = (end - start) * 1e-9  # nanoseconds to seconds
+    pulses = op.H.pulses
+
+    # The driven wires aren't the same as `op.wires` as `op.wires` contains
+    # all device wires due to interaction term.
+    op_wires = qml.wires.Wires.all_wires([pulse.wires for pulse in pulses])
+    mapped_wires = op_wires.map(wire_map)
+
+    frames = {w: device_frames[f"q{w}_drive"] for w in mapped_wires}
+    time_step = frames[0].port.dt * 1e9  # seconds to nanoseconds
+
+    pulse_sequence = PulseSequence().barrier(frames.values())
+
+    for pulse in pulses:
+        # Create waveform for each pulse in `ParametrizedEvolution`
+        if callable(pulse.amplitude):
+            amplitude = partial(pulse.amplitude, op.parameters[0])
+
+            # Calculate amplitude for each time step and normalize
+            amplitudes = onp.array([amplitude(t) for t in np.arange(start, end, time_step)])
+            amplitudes = amplitudes / onp.amax(amplitudes) * max_amplitude
```

Comment:
I dont think we should be necessarily multiplying by the maximum amplitude but rather a scaling factor and ensure at another point that this does not exceed the maximum amplitude (e.g. via `np.clip`). We currently dont have that scaling factor so I would put a dummy variable with value 1 here

Reply:
What is the motivation for normalizing? I would think if I make a gaussian pulse envelope with a max of 0.5, I want that to have half the amplitude of a pulse envelope with a max of 1.

Reply:
> I cant follow your example, can you explain?

Let's say I create a pulse for a duration of 15 ns, with a gaussian shaped amplitude that has a max of 1V. I am trying to synchronize it with something on another qubit that I've already tuned to be a pi-pulse, and I want them to have the same duration, so I want this to be a pi-half pulse that also has a duration of 15ns. 

Since I can't tune the duration, I will tune the voltage to implement the desired gate. Unfortunately, 1V max is driving with too much power, and I'm overshooting my desired rotation. I want to reduce the voltage so that I get a slower rotation, tuning it to provide me a 15ns pulse with my pre-defined shape that implements a 90 degree rotation. If my amplitude is always normalized, I can't do this; amplitude is not a tunable parameter.

Reply:
Thanks! Yep, I agree!